### DISCUSSION TITLE
[Feature] use multiple IP addresses when backing up the client.

### DISCUSSION THREAD

**Discussion by bai0012** [immich-app/immich#3258] [Feature Request]
### The feature

Translation to English:
I hope that immich's client can customize multiple IP addresses that can connect to the server-side, and then automatically detect the IP address that can connect and has the fastest speed during backup.

For example:
192.168.16.226 (local network address)
172.24.0.72 (local network address forwarded through ZeroTier)
154.3.x.x (public address)
and ipv6 address

### Platform

- [X] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #308
